tests/test_plugin_loader.py

```
@@ -2254,6 +2313,30 @@ def test_clone_or_update_repo_checkout_and_pull_branch(
             result = clone_or_update_repo(repo_url, ref, plugins_dir)
             self.assertTrue(result)
 
+    @patch("mmrelay.plugin_loader._is_repo_url_allowed", return_value=True)
+    @patch("mmrelay.plugin_loader._run_git")
+    def test_clone_or_update_repo_branch_pull_failure_force_syncs(
+        self, mock_run_git, _mock_is_allowed
```

Ruff (0.15.7)

[warning] Fixture `_mock_is_allowed` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)